kubernetes/ingress/supernovahunters.org.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: supernovahunters-org-tls
+  labels:
+    use-azuredns-solver: "true"
+spec:
+  issuerRef:
+    name: letsencrypt-prod
+    kind: ClusterIssuer
+  secretName: supernovahunters-org-tls
+  dnsNames:
+    - supernovahunters.org
+    - "*.supernovahunters.org"
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: supernovahunters.org
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
```

Comment:
remove this we don't use / need it. it's been copied from a template from an original one. By default nginx will pass on the paths without modification and we don't need to modify (rewrite) the paths for almost all our domains (if not all)

see for an example without the rewrite, https://github.com/zooniverse/panoptes/pull/3523/files
```suggestion
```